CONTRIBUTING.md

```diff
@@ -281,11 +282,21 @@ If you want to run a spell check of all `.Rmd` files locally, you can use run `R
 
 ## Rendering notebooks
 
+### Automatic rendering using GitHub actions
```

Comment:
Since this happens _after_ the PR has been accepted, we should probably move this section to after the part where we explain local rendering. This is a nice thing to know about, but if it is working properly, they should not run into it at all.

Reply:
See if you agree with where I moved it to.